src/components/CheckboxGroup/CheckboxGroup.js

```diff
@@ -0,0 +1,68 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { includes } from 'lodash';
+
+import { uniqueId } from '../../util/id';
+
+import Checkbox from '../Checkbox';
+
+/**
+ * A group of Checkboxes.
+ */
+const CheckboxGroup = ({
+  options,
+  onChange: onToggle,
+  value: activeValue,
+  name
+}) => (
+  <Fragment>
+    {options &&
+      options.map(({ label, value, className, ...props }) => (
+        <div key={value} className={className}>
+          <Checkbox
+            {...{ ...props, value, onToggle }}
+            checked={includes(activeValue, value)}
+            name={name || uniqueId('checkbox-group_')}
```

Comment:
Hm, I think this is actually a bug, because `uniqueId()` would get execute on each loop, creating a unique Id every time — but we want one common name for all checkboxes. Will fix.